.github/workflows/ci.yml

```diff
@@ -188,6 +188,16 @@ jobs:
           toolchain: 1.57.0
       - run: cargo check --lib --all-features
 
+  cross:
```

Comment:
Nit: Could this job be named more specifically (ie, 32-bit support) to be more immediately clear what it does?

<img width="1111" alt="image" src="https://github.com/rustls/webpki/assets/68570223/dba198fe-b500-4e3d-81f0-6f9a48c1bdfd">

Reply:
Added a more descriptive name, thanks!